cloud/azure/deploy/up.go

```diff
@@ -274,13 +273,10 @@ func (d *DeployServer) Up(request *deploy.DeployUpRequest, stream deploy.DeployS
 						ManagedIdentityID:             contEnv.ManagedUser.ClientId,
 						MongoDatabaseName:             mongodbName,
 						MongoDatabaseConnectionString: mongoConnectionString,
-						Config:                        euConfig.ContainerApps,
+						Config:                        *euConfig.ContainerApps,
 					}, pulumi.Parent(contEnv))
-					if err != nil {
-						return err
-					}
-				default:
-					return status.Errorf(codes.InvalidArgument, "Invalid target %s for execution unit %s", euConfig.Target, eu.Name)
+				} else {
+					return status.Errorf(codes.InvalidArgument, "Invalid target for execution unit %s", eu.Name)
```

Comment:
For consistency with other error messaging.

```suggestion
					return status.Errorf(codes.InvalidArgument, "unsupported target for function config %s", eu.Name)
```